src/screens/Election/ElectionApplication.js

```diff
@@ -187,19 +188,19 @@ class ElectionApplication extends Component {
 
```

Comment:
In reference to line 179, the `button` style is not being used. Delete it or create a new style.